base/server/python/pki/server/deployment/scriptlets/configuration.py

```diff
@@ -654,6 +655,31 @@ def spawn(self, deployer):
 
                 self.validate_system_certs(deployer, nssdb, subsystem)
 
+            elif len(subsystems) > 1:
+
+                for s in subsystems:
+
+                    if s.name == subsystem.name:
+                        continue
+
+                    logger.info('Importing sslserver cert from %s', s.type)
+                    subsystem.config['%s.sslserver.cert' % subsystem.name] = \
+                        s.config['%s.sslserver.cert' % s.name]
+
+                    logger.info('Importing subsystem cert from %s', s.type)
+                    subsystem.config['%s.subsystem.cert' % subsystem.name] = \
+                        s.config['%s.subsystem.cert' % s.name]
+
+                    logger.info('Importing sslserver cert request from %s', s.type)
+                    subsystem.config['%s.sslserver.certreq' % subsystem.name] = \
+                        s.config['%s.sslserver.certreq' % s.name]
+
+                    logger.info('Importing subsystem cert request from %s', s.type)
+                    subsystem.config['%s.subsystem.certreq' % subsystem.name] = \
+                        s.config['%s.subsystem.certreq' % s.name]
+
+                    break
```

Comment:
Question: Why are we breaking here? Does this mean we replace only 1 subsystem everytime? If so, can we get rid of the loop?

Reply:
Since we're only installing one subsystem at a time, we can stop as soon as the current subsystem is updated. The other subsystems already have the cert/request data populated when they were originally installed.